pyproject.toml

```diff
@@ -1,30 +0,0 @@
-[build-system]
```

Comment:
did you delete this because the configs here are already specified elsewhere? If not sure, maybe just keep it for now so we don't end up under-specific? Your call though.

Reply:
Fixing.